app/index.html

```diff
@@ -90,7 +94,7 @@
                         <a class="page-scroll" href="explore.html">Explore</a>
                     </li>
                     <li>
-                        <a class="page-scroll" href="login.html" onclick = "initalizeFirebase();firebase.auth().signOut()">Logout</a>
+                        <a class="page-scroll" href="login.html" onclick = "firebase.auth().signOut()">Logout</a>
```

Comment:
Why delete` initalizeFirebase()` here? Can it function well? This sentence is not inside ng-controller.